modules/ROOT/pages/7.0-release-notes.adoc

```diff
@@ -134,6 +134,22 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 // CCFR here.
 
+=== Autocomplete would sometimes cause corrupt data when starting during text composition. Autocomplete no longer starts during composition.
```

Comment:
This was changed so remove the `Autocomplete no longer starts during composition.` part.

```suggestion
=== Autocomplete would sometimes cause corrupt data when starting during text composition.
```

We did this fix first then that caused IMEs to not work at all then I did a follow up to make sure IMEs works with the autocompleter something that has been broken for years. So the details on the DOCs on that ticket was correct when it was written but not correct after my patch. Since this ticket is the one that we are referring for the customer I thought it would make sense to also keep this one around. Even though the later fix is more wide it fixed any IME even Andorid the original ticket was just about IME and Korean on desktop.